src/items.zig

```diff
@@ -903,6 +972,27 @@ pub const ItemStack = struct { // MARK: ItemStack
 		self.storeToZon(allocator, result);
 		return result;
 	}
+
+	pub fn fromBytes(reader: *BinaryReader) !ItemStack {
+		const amount = try reader.readInt(u16);
```

Comment:
Could use a varint, which would bring the cost of empty slots to 1 byte.

Reply:
Ehh, sure, except that I have no idea how they work and how to implement that

Reply:
Just use readVarint/writeVarint, nothing else to do.